sdk/communication/communication-chat/CHANGELOG.md

```diff
@@ -1,18 +1,65 @@
 # Release History
 
+## 1.5.0-beta.1 (Unreleased)
+
+### Breaking Changes
+
+- Updated `ChatAttachment` properties
+  - `url` becomes optional
+  - `contentType` is removed
+  - Renamed `AttachmentType` and its enums (`"teamsInlineImage" | "teamsImage" | "file"`) to `ChatAttachmentType` and `"image" | "file" | "unknown"`
+
+### Other Changes
+
+- Updated @azure/communication-signaling to 1.0.0-beta.23
+
 ## 1.4.0 (2023-12-04)
 
 ### Features Added
 
 - Added `ChatAttachment` object that contains properties for inline images incoming from Teams Chat Threads.
 - Updated to @azure/communication-signaling@1.0.0-beta.22 with inline image support
 
+## 1.4.0-beta.2 (2023-10-10)
+
+### Features Added
+
+- Updated @azure/communication-common to 2.3.0
+- Updated @azure/communication-signaling to 1.0.0-beta.21
+
 ## 1.3.2 (2023-08-24)
 
 ### Features Added
 
 - Updated to @azure/communication-signaling@1.0.0-beta.20. Added EUDB real-time notification support. `https://*.trouter.teams.microsoft.com` and `https://teams.microsoft.com` needs to be allowed for real-time notifications.
 
+## 1.4.0-beta.1 (2023-08-17)
```

Comment:
We're creating a new branch feature/communication-chat from main, which doesn't include the historical beta release records. This Pull Request aims to reintegrate the existing beta release history into feature/communication-chat. This will establish `feature/communication-chat` as the branch for future beta releases. In doing so, we're effectively merging the main branch back into our feature branch without resolving a lot of conflicts outside the `communication-chat` folder